app/scripts/modules/core/src/pipeline/manualExecution/ManualPipelineExecutionModal.tsx

```diff
@@ -289,7 +289,7 @@ export class ManualExecutionModal extends React.Component<IManualExecutionModalP
     const triggers = this.formatTriggers(pipeline && pipeline.triggers ? pipeline.triggers : []);
     let trigger: ITrigger;
     if (this.props.trigger) {
-      // Certain fields like correlationId will cause unexepcted behavior if used to trigger
+      // Certain fields like correlationId will cause unexepected behavior if used to trigger
```

Comment:
ooo thanks for fixing my typos 😄

Reply:
No good deed goes unpunished...sorry to be the bearer of bad news (s/unexepected/unexpected):
```suggestion
      // Certain fields like correlationId will cause unexpected behavior if used to trigger
```

Reply:
OMG I made it worse! Thanks @dbyron0, I'll fix this with my next Deck PR. So embarrassing :see_no_evil:

Reply:
don't be emabrrasde magige, we ali maek ytpos msomeitmes!